Integrations/integration-mcafeeDam.yml

```diff
@@ -199,28 +272,76 @@ script:
         case 'fetch-incidents':
             return JSON.stringify(fetchIncidents());
         case 'dam-get-alert-by-id':
-            return sendRequest(finalUrl, 'alert', 'HH$Id=' + encodeURIComponent(args.id)).HedgehogXmlApi.alert;
+            var jsonRes=sendRequest(finalUrl, 'alert', 'HH$Id=' + encodeURIComponent(args.id)).HedgehogXmlApi.alert;
+            if (jsonRes) {
+                var entryContext = {
+                };
+                md = alertToMd(jsonRes);
+                return {Type: entryTypes.note, Contents: jsonRes, ContentsFormat: formats.json, HumanReadable: md, ReadableContentsFormat: formats.markdown, EntryContext: {}};
+            } else {
+                md = '### no alerts found\n';
+                return {Type: entryTypes.note, Contents: null, ContentsFormat: formats.json, HumanReadable: md, ReadableContentsFormat: formats.markdown};
```

Comment:
there is a bug that if the contents entry is null, or missing, you get panic. Told Aviad this... @aviad - did you fix already? Will clean the code once done.